src/typeDefs/types.ts

```diff
@@ -120,10 +121,19 @@ export const types = gql`
     attendeesCheckInStatus: [CheckInStatus!]!
     admins(adminId: ID): [User]
     status: Status!
+    projects: [EventProject]
     feedback: [Feedback!]!
     averageFeedbackScore: Float
   }
 
+  type EventProject {
+    _id: ID!
+    title: String!
+    description: String!
+    event: Event!
```

Comment:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it